Build/Projects/MonoGame.Framework.definition

```diff
@@ -42,10 +42,10 @@
       <Platform Name="Android">TRACE;ANDROID;GLES;OPENGL;OPENAL</Platform>
       <Platform Name="iOS">IOS;GLES;OPENGL;OPENAL</Platform>
       <Platform Name="tvOS">IOS;TVOS;GLES;OPENGL;OPENAL</Platform>
-      <Platform Name="Linux">TRACE;LINUX;OPENGL;OPENAL;DESKTOPGL</Platform>
+      <Platform Name="Linux">TRACE;LINUX;OPENGL;OPENAL;DESKTOPGL;SUPPORTS_EFX</Platform>
       <Platform Name="MacOS">MONOMAC;OPENGL;OPENAL</Platform>
```

Comment:
EFX was previously wrapped in a !MONOMAC define.  I had SUPPORTS_EFX in that line before, but that line defines MONOMAC, and that is what caused the last build failure.